week-3/C-array-every/exercise.js

```diff
@@ -6,7 +6,9 @@
 var students = ["Omar", "Austine", "Dany", "Swathi", "Lesley", "Rukmini"];
 
 // -- Complete this function -->
-function hasOnlyStudents() {}
+function hasOnlyStudents(mentor, student) {
+  return students === group;
+}
```

Comment:
Not quite sure what you intended here. The function `hasOnlyStudents` has two parameters but you don't use either of them? Check the readme for the example. It wants you to use the `every` method. The  functionality could be summarised as "for every member of the group array, check that the name is included in the students array".